tests/mocks/playwright-mocks.ts

```diff
@@ -1,4 +1,4 @@
-import { BrowserContext } from 'playwright-core';
+import { BrowserContext } from 'playwright';
```

Comment:
Shouldn't this now be `@playwright/test`?

Reply:
My reasoning here was that given the existing tests use the `playwright` rather than `@playwright/test`, the lib seemed like the reasonable alternative so as to be in line with the rest of the existing test code. Otherwise, this file will be the only one in the existing test codebase that pulls from the new `@playwright/test`.